### ISSUE TITLE
singa_easy/modules/mod_gmreg/gm_prior_optimizer_pytorch.py", line 235 divided by zero

### ISSUE THREAD

**Issue by delphieritas** [nusdbsystem/singa-easy#60]
Bug location:
https://github.com/nusdbsystem/singa-easy/blob/04ed0d979ea700d6662c6e395da29336f75dc3da/singa_easy/modules/mod_gmreg/gm_prior_optimizer_pytorch.py#L235

------------------
Error message:
Epoch 1/50
Got batch_idx and batchdata 0
zero the optimizer
THCudaCheck FAIL file=/pytorch/aten/src/THC/THCGeneral.cpp line=405 error=11 : invalid argument
..../singa_easy/modules/mod_gmreg/gm_prior_optimizer_pytorch.py:235: RuntimeWarning: divide by zero encountered in true_divide
  float(normalization_coefficient))).float()
step:  0
name:  features.0.weight
data grad norm:  0.23468745
reg_grad_w_dev l2 norm:  inf
delta w norm:  inf

---------------------
Error condition:
Cause meaningless epoch_accuracy and 'NaN' loss

**Comment by chrishkchris** [comment]
https://github.com/nusdbsystem/singa-easy/blob/04ed0d979ea700d6662c6e395da29336f75dc3da/singa_easy/modules/mod_gmreg/gm_prior_optimizer_pytorch.py#L229

At line 229, normalization_coefficient is the labelnum * trainnum

trainnum is the number of sample in the training dataset, the labelnum is the number of class in classification (e.g. cifar10 has 10 classes)

@delphieritas If normalization_coefficient is 0, this means that the labelnum and trainnum are not fed correctly, please check.

**Comment by delphieritas** [comment]
https://github.com/nusdbsystem/singa-easy/pull/64
